kms_log/main.tf

```diff
@@ -10,11 +10,18 @@ data "aws_kms_key" "application"
     key_id = "alias/${var.env_name}-login-dot-gov-keymaker"
 }
 
+
+data "aws_s3_bucket" "lambda"
```

Comment:
Since each environment is going to call this module separately, we probably don't want this bucket redefined in each one.

Can we make this a variable that's passed in by the caller?

Reply:
It looked to me like we were using a single bucket per account for the lambdas.  This is only a data block, not creating the bucket.

Reply:
You're right, I see how it works now.